src/components/tag-command.tsx

```diff
@@ -20,6 +21,12 @@ function isTagArgv(argv: {readonly _: unknown[]}): argv is TagArgv {
   return argv._[0] === `tag`;
 }
 
+function createTag(tag: string): Tag {
+  const [key = tag, value] = tag.split(`=`);
```

Comment:
Do you need the fallback to tag?
If tag does not contain '=' key should already be tag, or what am i missing?

Reply:
That's true, but typescript does not know that this array always contains at least one element (`noUncheckedIndexedAccess` is enabled), and because of the destructuring and short-hand usage below there was no good place to cast this or use a nonNullAssertion.

Reply:
But I could cast the array. 🤔

Reply:
fixed